### PR TITLE
Add new trees to Woodcutting and a new cook message to Cooking Crowdsourcing

### PR DESCRIPTION
**Pull request by pwatts6060**
Add more data to track for crowdsourcing woodcutting and cooking. New log object ids, log messages, and a spit-roast cooking message.

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/crowdsourcing/woodcutting/CrowdsourcingWoodcutting.java [hunk 1]
```diff
@@ -55,57 +55,123 @@
...
+			add(ObjectID.TEAK).
+			add(ObjectID.TEAK_15062).
+			add(ObjectID.MAGIC_TREE_10834).
+			add(ObjectID.MAGIC_TREE_8409).
+			add(ObjectID.HOLLOW_TREE_10821).
+			add(ObjectID.HOLLOW_TREE_10830).
+			add(ObjectID.ACHEY_TREE).
+			add(ObjectID.REDWOOD).
+			add(ObjectID.REDWOOD_29670).
+			add(34633, 34635, 34637, 34639). //redwood tree patch farming guild
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/crowdsourcing/woodcutting/CrowdsourcingWoodcutting.java:134]: Use the variables found in NullObjectID

File: runelite-client/src/main/java/net/runelite/client/plugins/crowdsourcing/woodcutting/CrowdsourcingWoodcutting.java [hunk 2]
```diff
@@ -55,57 +55,123 @@
...
-		add(ObjectID.JUNGLE_TREE_15948).
-		add(ObjectID.MAPLE_TREE_10832).
-		add(ObjectID.MAHOGANY).
-		add(ObjectID.TEAK).
-		add(ObjectID.MAGIC_TREE_10834).
-		add(ObjectID.HOLLOW_TREE_10821).
-		add(ObjectID.HOLLOW_TREE_10830).
-		add(ObjectID.ACHEY_TREE).
-		build();
+			add(ObjectID.OAK).
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/crowdsourcing/woodcutting/CrowdsourcingWoodcutting.java:58]: Indentation needs to be fixed in this file. All these changed lines should have two leading tabs, not three.

**Review by Nightfirecat — CHANGES_REQUESTED** [review]
@leejt to review

**Review by Nightfirecat — COMMENTED** [review]
LGTM

**Comment by Nightfirecat** [comment]
@leejt Please review this PR

**Comment by leejt** [comment]
For the farming trees, the MenuOptionClicked's ID will always be that of the base patch, rather than the tree it's been transformed into. I'd suggest removing the farming-related IDs instead of trying to wrangle the ID out of it (especially because farming trees are particularly weird to begin with, and they probably aren't the primary source of redwood data anyway).

Cooking changes LGTM - it might be nice to figure out the spider and snail messages too.

**Comment by pwatts6060** [comment]
> For the farming trees, the MenuOptionClicked's ID will always be that of the base patch, rather than the tree it's been transformed into. I'd suggest removing the farming-related IDs instead of trying to wrangle the ID out of it (especially because farming tree tick mechanics are particularly weird to begin with, and they probably aren't the primary source of redwood data anyway).
> 
> Cooking changes LGTM - it might be nice to figure out the spider and snail messages too.

Ok I removed the redwood farming tree objects.

I checked the farming messages for snails and spiders and the current stuff already covers it I think. 
`|| message.startsWith("You accidentally burn")`
"You accidentally burn the spider."
"You accidentally burn the snail."

`|| message.startsWith("You cook")`
"You cook a spider."
"You cook the fat snail."
"You cook the lean snail."
"You cook the thin snail."

**Pull request merged**